src/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+
+function Title() {
+  return <div>Counter</div>;
+}
+
+function ButtonClickme(props) {
+  const { value, onClick } = props;
+  return (
+    <button type="button" onClick={onClick}>
+      Click me!
+      (
+      {value}
+      )
+    </button>
+  );
+}
+
+function Button(props) {
+  const { value, onClick } = props;
```

Comment:
이렇게 처리해줘도 될 것 같아요 ㅎㅎ
```suggestion
function Button({ value, onClick }) {
```